app/forms/user_form.rb

```diff
@@ -0,0 +1,26 @@
+class UserForm < Form
+  FIELDS = %i[
+    first_name
+    last_name
+    email
+    id
+  ].freeze
+
+  attr_accessor(*FIELDS)
+
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :email, presence: true
+  validates :email, email_address: true
+  validate :email_is_lowercase
```

Comment:
I don't think we need to validate if the email is lowercase since `SimonWhatley@example.com` or `SIMONWHATLEY@EXAMPLE.COM` is a valid email.
```suggestion

```

Reply:
Removed here: https://github.com/DFE-Digital/publish-teacher-training/pull/3159/commits/c31f632d7c172e292dbce1e6833394f25654d664

And converting to lowercase prior to saving here: https://github.com/DFE-Digital/publish-teacher-training/pull/3159/commits/8165ef76991f0dc7a0a043ccf80de0016bc3f60f